openwisp-qa-check

```diff
@@ -60,6 +68,26 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  stylelint $(find . -type f -name "*.css") &&
+    echo "SUCCESS: Stylelint check successful!" ||
+    {
+      echoerr "ERROR: Stylelint check failed! Hint: did you forget running openwisp-qa-format?"
+      FAILURE=1
+    }
+
+}
+
+runjshintcheck() {
+  jshint --verbose $(find . -type f -name "*.js" -a ! -path "*vendor/*.js") &&
```

Comment:
what happens if `jshint` is not installed? Can you make sure the function skips with a warning?
We need this to avoid breaking existing builds which are using the master version of this module.

Reply:
Added a check for this